src/app/app-interactive-map/components/app-reaction/components/app-panel/app-panel.component.ts

```diff
@@ -28,23 +29,15 @@ export class AppPanelComponent {
   @Input() public title: string;
   @Input() public type: string;
   @Input() public placeholder: string;
+  @ViewChild('queryInput') queryInput: ElementRef;
 
   public querySearch: FormControl = new FormControl();
-  public reactions: Reaction[] = [{'bigg_id': 'FK', 'name': 'Fucokinase', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FT', 'name': 'Trans,trans,cis-geranylgeranyl diphosphate synthase', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FCI', 'name': 'L-fucose isomerase', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FHL', 'name': 'Formate-hydrogen lyase', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FUM', 'name': 'Fumarase', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FDH', 'name': 'Formate dehydrogenase', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FRD', 'name': 'FRD', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FQR', 'name': 'Cyclic Electron Flow', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FTR', 'name': 'Ferredoxin thioredoxin reductase', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FBA', 'name': 'Fructose-bisphosphate aldolase', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'F4D', 'name': 'F4D', 'model_bigg_id': 'Universal', 'organism': ''},
-      {'bigg_id': 'FBP', 'name': 'Fructose-bisphosphatase', 'model_bigg_id': 'Universal', 'organism': ''}];
+  public reactions: Reaction[] = [];
```

Comment:
I feel like this could be a stream (Observable / Subject), that we pass down to components.

This should rather be a subject, which is composed of the result of the http request in 
https://github.com/DD-DeCaF/caffeine/pull/38/files#diff-54003de609ef54825ca0fd358a232e85R87
And you could send in a null object in 
https://github.com/DD-DeCaF/caffeine/pull/38/files#diff-54003de609ef54825ca0fd358a232e85R81